test/unit/interface.js

```diff
@@ -30,9 +30,13 @@ describe('interface', function () {
     }
 
     _interface(SomeClass);
+
     const fakeInstance = new SomeClass();
     chai.expect(fakeInstance[Symbol.toStringTag]).to.eq('object');
-    fakeInstance._asyncStack = ['line1', 'line2', 'line3'];
+    fakeInstance._asyncStack = {
```

Comment:
How difficult would it be to create a less synthetic unit test (e. g. for an in-memory SQLite db instance) that would have an actual asyncStack?

Reply:
Less synthetic was already in tests here https://github.com/istarkov/knex/blob/master/test/integration/builder/additional.js#L1127
There was no need to change it. As output is the same.

Reply:
interface.js also is not used as I see in tests at all, probably need to be removed at all

Reply:
See test/db-less-test-suite.js:

```
  // require('./unit/interface'); ToDo Uncomment after fixed
```

Can you uncomment it?

Reply:
No, it didnt work before, it doesnt work now. I have no idea why and I don't have enough knowledge why it was commented and not working at all. Probably standalone PR must be created by someone smart to fix it.